circle.yml

```diff
@@ -13,7 +13,12 @@ test:
 #   - npm run lint
     - npm test
     - npm run test-mocha
-    - npm run integration
+    - ./node_modules/.bin/integration-loader
```

Comment:
Maybe `loader` and `setup` can be combined into one step, and the tests can just be split into ILP Kit and non-ILP Kit